<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionWidget</class>
 <widget class="QWidget" name="QuestionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>155</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>155</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">QuestionWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="text_answer">
     <property name="toolTip">
      <string>Your answer to this question</string>
     </property>
     <property name="documentTitle">
      <string/>
     </property>
     <property name="plainText">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="button_answer">
       <property name="text">
        <string>&amp;Answer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_delete">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="label_question">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">[Question]</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_user_asked">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;strong&gt;Anonymous&lt;/strong&gt; asked:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="chk_post_to_twitter">
     <property name="text">
      <string>Post to Twitter</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
